modules/serverless-list-source-types.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * serverless-kn-source.adoc
+
+[id="serverless-list-source-types_{context}"]
+= Listing available event source types using `kn`
+
+You can list the available event source types in the terminal by using the following command:
+----
+$ kn source list-types
+----
+
+The default output for this command will look like:
+----
+TYPE              NAME                                            DESCRIPTION
+ApiServerSource   apiserversources.sources.knative.dev            Watch and send Kubernetes API events to a sink
+PingSource        pingsources.sources.knative.dev                 Send periodically ping events to a sink
+SinkBinding       sinkbindings.sources.knative.dev                Binding for connecting a PodSpecable to a sink
+----
+
+It is also possible to list available event source types in JSON format:
```

Comment:
Perhaps you wanted to say YAML. The command says yaml.

Reply:
This actually came from https://github.com/knative/client/blob/master/docs/cmd/kn_source_list-types.md#examples
@rhuss should this be YAML or should the command be 
`kn source list-types -o JSON`
or maybe we want to include both? I'm not sure which if any of these are implemented. Thanks!

Reply:
thats a typo in upstream docs https://github.com/knative/client/blob/master/docs/cmd/kn_source_list-types.md#examples 
fixing it..

Reply:
OK. So it should be YAML in the comment, according to https://github.com/knative/client/pull/684/files . Thanks:)